OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -101,7 +101,8 @@ IEnumerable<object> IActorPreviewInitInfo.ActorPreviewInits(ActorInfo ai, ActorP
 			yield return new FacingInit(PreviewFacing);
 		}
 
-		[Desc("Condition when this aircraft should land as soon as possible and refuse to take off.")]
+		[Desc("Condition when this aircraft should land as soon as possible and refuse to take off. "
+			+ "This only applies while the aircraft is above terrain which is listed in LandableTerrainTypes.")]
```

Comment:
nit: the convention is to use a comma since the constructor takes a `param` list of `string`s.